ballerina/build.gradle

```diff
@@ -849,7 +840,16 @@ task testExamples() {
          'testerina-group-tests',
          'testerina-mocking-objects',
          'testerina-mocking-functions',
-         'json-record-map-conversion'
+         'json-record-map-conversion',
+         'http-timeout',
+         'http-retry',
+         'http-failover',
+         'http-load-balancer',
+         'http-circuit-breaker',
```

Comment:
Why are we ignoring these ? When I last checked, these tests were passing. Is there any issues now?

Reply:
Yes, those were failed in this PR. That's why I added to ignoreList

Reply:
I don't get why, these were fixed recently. I just tried locally, and they were passing. Any specific changes to them to fail?

Reply:
Actually the tests are failing, not the BBE.

Reply:
Exactly my question. These tests were fixed very recently, and I checked locally, with the latest ballerina-lang build, and they passed.

Reply:
```sh
Running Tests
        buddhi/http-timeout:0.1.0
[ballerina/http] started HTTP/WS listener 0.0.0.0:9090
[ballerina/http] started HTTP/WS listener 0.0.0.0:8080
[ballerina/http] stopped HTTP/WS listener 0.0.0.0:9090
[ballerina/http] stopped HTTP/WS listener 0.0.0.0:8080

        [fail] testFunc:
            Assertion Failed!: expected 'Request timed out. Please try again in sometime.' but found '<html><head><meta http-equiv='refresh' content='1;url=/login?from=%2Fhello'/><script>window.location.replace('/login?from=%2Fhello');</script></head><body style='background-color:white; color:white;'>
                Authentication required
                <!--
                You are authenticated as: anonymous
                Groups that you are in:
                
                Permission you need to have (but didn't): hudson.model.Hudson.Read
                ... which is implied by: hudson.security.Permission.GenericRead
                ... which is implied by: hudson.model.Hudson.Administer
                -->
                </body></html>                                                                                                                                                                                                                                                                                                            '
                at ballerina.test.0_0_0:createBallerinaError(assert.bal:41)
                ballerina.test.0_0_0:assertEquals(assert.bal:82)
                buddhi.http-timeout.0_1_0.tests.http_timeout_test:testFunc(tests/http_timeout_test.bal:13)
```

Reply:
Isn't this another issue?

Reply:
I think so. I just enabled it, let's check the build log after it completed.

Reply:
Approving this since the checks are passing.

Reply:
My mistake. I think this is an issue in my local machine.